src/main/kotlin/io/provenance/client/grpc/PbClient.kt

```diff
@@ -75,11 +75,11 @@ class PbClient(
     val paramsClient = cosmos.params.v1beta1.QueryGrpc.newBlockingStub(channel)
     val slashingClient = cosmos.slashing.v1beta1.QueryGrpc.newBlockingStub(channel)
     val stakingClient = cosmos.staking.v1beta1.QueryGrpc.newBlockingStub(channel)
+    val tendermintClient = cosmos.base.tendermint.v1beta1.ServiceGrpc.newBlockingStub(channel)
```

Comment:
Technically, this group is for query clients while the previous section is for service clients. It doesn't super matter, they were just grouped for style/organization. Whichever you think is a a good location is fine with me.

Reply:
@vwagner I was wondering about that. Thanks for pointing that out. I'll make it a service client.